ext/django_stubs_ext/settings.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+from typing import Any, TypedDict, type_check_only
+
+from typing_extensions import NotRequired
+
+
+@type_check_only
+class _TemplatesSetting(TypedDict):
```

Comment:
Should we keep the leading `_` for symbols we expose in `django_stubs_ext` ?

Reply:
This probably needs to be documented somewhere too, should I add details in the ext/README.md and link to it in the main readme ?

Reply:
I think that this is a separate feature. We need to rework the whole `ext/README.md` file. It is very old and does not cover many use-cases.